src/test/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/BootstrapClusterStepTest.java

```diff
@@ -160,7 +164,11 @@ public void testExecuteScriptFiles()
 
         step.execute(config);
 
-        verify(step, times(2)).validateFile(anyString());
+        ArgumentCaptor<Path> pathCaptor = ArgumentCaptor.forClass(Path.class);
+        verify(step, times(2)).parseScript(any(ElasticsearchClient.class), eq(log), pathCaptor.capture());
+        List<Path> capturedPaths = pathCaptor.getAllValues();
```

Comment:
Please add an assert after this line `assertEquals(2, capturedPaths.size())`
then before 170 add a `assertNotNull(capturedPaths.get(0))`
and then before 171 a similar assert not null for the capturedPaths.get(1)

That way, it would be more obvious what's wrong.

Reply:
Done, also did the same for `testExecuteJsonFiles`